standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1905,6 +1905,8 @@ public enum ConfVars {
         "Only collect the non-native table's summary that has been updated/changed in configured recent days."),
     METADATA_SUMMARY_MAX_NONNATIVE_TABLES("hive.metatool.summary.maxNonNativeTables", "hive.metatool.summary.maxNonNativeTables", "",
         "The maximum non-native tables allowed per table type during collecting the summary."),
+    METADATA_SUMMARY_NONNATIVE_THREADS("hive.metatool.summary.nonnative.threads", "hive.metatool.summary.nonnative.threads", 20,
+        "Number of threads to be allocated for MetaToolTaskMetadataSummary for collecting the non-native table's summary."),
```

Comment:
I feel the thread count is too high for an operation that would run in the background and that is not run very often. Thoughts?